File: Haskell/beginner/1143.hs
-- https://judge.beecrowd.com/pt/problems/view/1143
-- Quadrado e ao Cubo

-- Importing intercalate
import Data.List (intercalate)

-- Main body
main :: IO ()
main = do
    usr_value <- readLn :: IO Int
    let usr_return = ([(x, x*x, x*x*x )| x <- [1..usr_value]])
    let results = intercalate "\n" (map (\(x,y,z) -> show x ++ " " ++ show y ++ " " ++ show z) usr_return)
    putStrLn results

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1143.c
// https://judge.beecrowd.com/pt/problems/view/1143
// Quadrado e ao Cubo

// Libraries
#include <stdio.h>

// Main
int main(){
    // Starting code, declarating variables
    int usr_input;

    // Scanning user input
    scanf("%d", &usr_input);

    // Iterating
    for(int i = 1; i  < usr_input + 1; i++){
        // Showing results
        printf("%d %d %d\n", i, i * i, i * i * i);
    }
    // Ending Code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandoserra